.github/workflows/test.yml

```
@@ -60,6 +60,6 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
       - name: Run unit tests
-        run: npx jest
+        run: yarn unit:win
```

If we need a script only in the single place, let’s put command here directly.

```suggestion
        run: npx uvu -r tsm test ^.*test.([tj]s)
```